src/utils/Processing.ts

```diff
@@ -240,6 +241,29 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+export function checkNullValuesOnArray(resource: any, parentName = ''): void {
+  const resourceName = parentName ? parentName : resource.id ?? resource.name;
+  for (const property in resource) {
+    // We'll want to exclude properties beginning with an "_" as those are arrays containing extensions
+    // corresponding to a primitive array, per the FHIR spec
+    if (property.startsWith('_')) continue;
+    if (_.isPlainObject(resource[property]))
+      // If we encounter an object property, we'll want to check its properties as well
+      checkNullValuesOnArray(resource[property], resourceName);
+    if (Array.isArray(resource[property])) {
+      const nullIndexes: number[] = [];
+      resource[property].forEach((element: any, index: number) => {
+        if (element === null) nullIndexes.push(index);
```

Comment:
If the array item not null, and it's a plain object, we need to check that object for array properties w/ null values too, right?  E.g.,
```json5
{
  "foo": [
    {
      "coding": [
        null, // this is bad
        { "system": "http://foo.org", "code": "bar" }
      ]
    }
  ] 
}
```
The `foo` array is ok, but the `foo[0].coding` array is not ok.